internal/commands/app/diff_test.go

```diff
@@ -139,26 +135,100 @@ Removed Dependencies
   - debug@4.3.1
 Modified Dependencies
   * underscore@1.9.1 -> underscore@1.9.2
-`, out.String())
+`
+		t.Run("with include node modules set it should diff function dependencies", func(t *testing.T) {
```

Comment:
we actually intentionally are avoiding things like that in the test descriptions for readability purposes

when you have something like `--` (double dash) in the test description, in the `go test` output it will be replaced with two underscores, and it starts to make the tests harder to read as they cascade out to the terminal

I also prefer to avoid "code words" where possible in the test descriptions, b/c those can change and these tests can become stale pretty quickly

i'd leave this as is

Reply:
ah ok, my bad! thanks for the context @makesitgo!